src/helpers/_font-faces.scss

```diff
@@ -21,47 +21,26 @@
 
       @font-face {
         font-family: "nta";
-        src: govuk-font-url("light-2c037cf7e1-v1.eot");
-        src: govuk-font-url("light-2c037cf7e1-v1.eot?#iefix") format("embedded-opentype"),
-             govuk-font-url("light-f38ad40456-v1.woff2") format("woff2"),
-             govuk-font-url("light-458f8ea81c-v1.woff") format("woff");
+        src: govuk-font-url("light-df2b968d71-v2.eot");
+        src: govuk-font-url("light-df2b968d71-v2.eot?#iefix") format("embedded-opentype"),
+             govuk-font-url("light-68f0c84f0e-v2.woff2") format("woff2"),
+             govuk-font-url("light-222368e53d-v2.woff") format("woff");
         font-weight: normal;
         font-style: normal;
         font-display: fallback;
```

Comment:
Matt recommended to change this to `swap` in his initial work

Reply:
Yes this still looks like the recommended setting. Google Fonts now allows passing in the `display` parameter and it by default is set to `swap`:

[https://www.zachleat.com/web/google-fonts-display/](https://www.zachleat.com/web/google-fonts-display/)